glue/__init__.py

```diff
@@ -118,6 +118,11 @@ def make_selector_func(roi):
     from .qt.custom_viewer import CustomViewer
     return CustomViewer.create_new_subclass(name, **kwargs)
 
+# Load built-in plugins
+from .config import load_plugins
+load_plugins()
```

Comment:
Cool. In my mind the priorities are (in decreasing order):

1) Anything which relies on import for auto-discovery (like the collection of available plugins, data viewers, whatever) gets imported by the time it's first used.
2) Such modules are imported as late as possible, so that tasks like  `glue --version`, `from glue import core`, run quickly
3) The two tasks above run if Qt isn't installed